client/src/controllers/MessagesController.ts

```diff
@@ -0,0 +1,77 @@
+import { Controller } from '@hotwired/stimulus';
+
+/**
+ * Adds the ability for a controlled element to pick messages from template,
+ * add/stack (depending upon clear:true/false) them within the provided unordered list container
+ * when called.
+ *
+ * @example
+ * <div data-controller="w-messages"
+ *      data-action="w-messages:add@document->w-messages#add"
+        data-w-messages-added-class="new"
+        data-w-messages-show-class="appear"
+ * >
+      <ul data-w-messages-target="messages"></ul>
+      <template data-w-messages-target="templates">
+        <li data-message-status="error/success"><span></span></li>
+      </template>
+ * </div>
+ */
+export class MessagesController extends Controller<HTMLFormElement> {
+  static classes = ['added', 'show'];
+  static targets = ['messages', 'templates'];
+  static values = {
+    showDelay: { default: 100, type: Number },
+  };
+
+  declare readonly hasMessagesTarget: boolean;
+  declare readonly hasAddedClass: boolean;
+  declare readonly hasShowClass: boolean;
+  declare readonly hasTemplatesTarget: boolean;
+  declare messagesTarget: HTMLElement;
+  declare readonly addedClass: string;
+  declare readonly showClass: string;
+  declare readonly showDelayValue: number;
+  declare templatesTarget: HTMLTemplateElement;
```

Comment:
```suggestion
  declare readonly addedClass: string;
  declare readonly containerTarget: HTMLElement;
  declare readonly hasAddedClass: boolean;
  declare readonly hasShowClass: boolean;
  declare readonly showClass: string;
  declare readonly templatesTarget: HTMLTemplateElement;

  declare showDelayValue: number;
```

A few clean up items;
* use `container` (as per other notes) instead of `templates` for the target
* `value` items will be not read only, all other items should be read only
* Re-ordering via alpha sort